tests/epyccel/test_kind.py

```diff
@@ -40,10 +41,39 @@ def add_real(a, b):
         c = a+b
         return c
 
-    epyc_add_real = epyccel(add_real, fflags="-Werror -Wconversion-extra")
+    epyc_add_real = epyccel(add_real , fflags="-Werror -Wconversion-extra")
 
     assert(add_real(1.0,2.0)==epyc_add_real(1.0,2.0))
 
+def test_output_types_1(language):
+    @types('float32')
+    def cast_to_int(a):
+        b = int(a)
+        return b
+
+    f = epyccel(cast_to_int, language = language)
+    assert(type(cast_to_int(5.2)) == type(f(5.2)))
+
+def test_output_types_2(language):
+    @types('int')
+    def cast_to_float(a):
+        b = float(a)
+        return b
+
+    f = epyccel(cast_to_float,language= language)
+    assert(type(cast_to_float(5)) == type(f(5)))    
```

Comment:
This type of check raises an error in Codacy, although it makes sense in this context (comparing types of objects is rare, usually you test if the type is something specific, so codacy is not built for this). You can disable the codacy test for this line by adding the following comment to the end of the line:
```
 # pylint: disable=unidiomatic-typecheck
```